app/views/govuk_publishing_components/components/_contextual_guidance.html.erb

```diff
@@ -0,0 +1,25 @@
+<%
+  html_for ||= nil
```

Comment:
Should this attribute actually be required? I'm not sure this does anything without it.

Reply:
I've updated this to not have a default value and added a test to make sure it fails if `html_for` is not provided.